index.d.ts

```diff
@@ -1,3 +1,5 @@
 declare module 'ember-window-mock' {
   export default window;
+  export function setupWindowMock(hooks: {afterEach: (fn: Function) => void}): void;
```

Comment:
```suggestion
  export function setupWindowMock(hooks: {afterEach: (fn: () => void) => void}): void;
```
is a better type for the callback because the reset function
https://github.com/kaliber5/ember-window-mock/blob/ebdebe5705b2b9baeb429957177f22e47c903439/addon-test-support/-private/window.js#L62-L66
you are passing to afterEach
https://github.com/kaliber5/ember-window-mock/blob/ebdebe5705b2b9baeb429957177f22e47c903439/addon-test-support/-private/setup-window-mock.js#L10
takes no arguments and does not return anything. This should be compatible with both qunit and mocha types.

Reply:
Thanks.
I've updated the PR based on the feedback.